### PR TITLE
REST API

### LINKED ISSUE #41: Introduce packit service REST API
@james-antill would like to start working on a frontend for packit-service. The frontend should be a separate project (which would live in a new repo). In order to do such a thing, we'll need to have a REST API where the frontend would access data from packit service.

Here's a list of things which the service should serve over its API:
* a list of jobs
  * ID
  * status
  * type
  * project
  * username who triggered it
* logs
* job details
  * a link to objects/aritfacts: e.g. copr build, koji builds, dist-git PR, commits...

feel free to add more

Step 1: start storing data inside redis in a structural way and create a solid python interface for it #57
Step 2: create the API

### PR DESCRIPTION
**Pull request by jpopelka**
Fixes #41

### CODE REVIEW

File: packit_service/service/api/healthz.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+@ns.route("/")
+class HealthCheck(Resource):
+    def get(self):
+        return jsonify({"msg": "We are healthy!"})
+
+    def head(self):
+        return self.get()
+
+    def post(self):
```
Comment (TomasTomecek) [inline packit_service/service/api/healthz.py]: I understand head and get, but post? I'd say it's fine if the webserver responds with != 200 for post on /healthz

Comment (jpopelka) [reply packit_service/service/api/healthz.py]: Yeah, I don't understand it either :-D  https://github.com/packit-service/packit-service/commit/5937c5ba056f3a97970935e3f953acaa32a15bd3

Comment (TomasTomecek) [reply packit_service/service/api/healthz.py]: :D

Comment (TomasTomecek) [reply packit_service/service/api/healthz.py]: no idea what I was thinking back then

**Pull request merged**